templates/media/media-with-message.php

```diff
@@ -0,0 +1,161 @@
+	</span>
+	<div id="rtm-media-gallery-uploader" class="rtm-media-gallery-uploader">
+		<?php
+		rtmedia_uploader(
+			[
+				'is_up_shortcode' => false,
+				'allow_anonymous' => true,
+				'privacy_enabled' => false,
+			]
+		);
+			?>
+	</div>
+	<input type="hidden" id="rtm_bpm_uploaded_media" name="rtm_bpm_uploaded_media" />
+	<?php
+}
+
+/**
+ * Add additional parameter media ID in BuddyPress message SEND MESSAGE process.
+ * And Insert data into MEDIA_META table.
+ *
+ * @param object $message Getting parameter response when sending message.
+ **/
+function rtm_add_message_media_params( $message ) {
+	$insert_media_object  = new RTDBModel( 'rtm_media_meta' );
+	$message->media_array = filter_input( INPUT_POST, 'rtm_bpm_uploaded_media' );
+	$media                = explode( ',', $message->media_array );
+	if ( ! empty( $media ) && null !== $media ) {
+		foreach ( $media as $media_id ) {
+			$insert_media_object->insert(
+				[
+					'media_id'   => $media_id,
+					// Adding meta data into custom meta table.
+					'meta_key'   => 'rtm-bp-message-media', // phpcs:ignore
+					// Adding meta data into custom meta table.
+					'meta_value' => $message->id, // phpcs:ignore
+				]
+			);
+		}
+	}
+	?>
+	<script>
+		jQuery( '#msg-success-bp-msg-media' ).hide();
+		jQuery( '.rtm-media-msg-upload-button' ).attr( 'id', 'rtm_show_upload_ui' );
+		jQuery( '.rtm-media-msg-upload-button' ).html( '' );
+		jQuery( '.rtm-media-msg-upload-button' ).html( 
+			jQuery( '<i>',
+				{ class: 'dashicons dashicons-upload rtmicon' }
+			) );
+		jQuery( '#rtm_show_upload_ui' ).append( 'Upload Media File' );
+	</script>
+	<?php
+}
+
+/**
+ * As a result show attached media with message.
+ **/
+function show_rtm_bp_msg_media() {
+	$get_data_object = new RTDBModel( 'rtm_media_meta' );
+	$media_result    = $get_data_object->get( [ 'meta_value' => bp_get_the_thread_message_id() ] );  // phpcs:ignore
+	$url             = explode( 'messages/', sanitize_text_field( wp_unslash( filter_input( INPUT_SERVER, 'REQUEST_URI' ) ) ) );
+	if ( '0' !== $media_result[0]->media_id ) {
+	?>
+	<ul class='rtmedia-list-media rtm-gallery-list clearfix' style = 'margin-top: 10px;'>
+	<?php
+	foreach ( $media_result as $media_result_array_value ) {
+		$media     = rtmedia_image( 'rt_media_thumbnail', $media_result_array_value->media_id, false );
+		$media_url = $url[0] . '/media/' . $media_result_array_value->media_id . '/';
+		?>
+
+			<li class='rtmedia-list-item rtmedia_bp_msg_media_upload_render' id="<?php echo $media_result_array_value->media_id; // @codingStandardsIgnoreLine?>">
+				<a href="<?php echo esc_url( $media_url ); ?>" class="<?php echo esc_attr( apply_filters( 'rtmedia_gallery_list_item_a_class', 'rtmedia-list-item-a' ) ); ?>">
+					<div class="rtmedia-item-thumbnail">
+						<img src="<?php echo esc_url( $media ); ?>" alt="<?php echo esc_attr( apply_filters( 'rtmc_change_alt_text', $alt_text, $rtmedia_media ) ); ?>">
```

Comment:
Don't use `apply_filters()` within some function. It should be in new line with proper documentation. Save the result in a variable and then use it.